CMakeLists.txt

```diff
@@ -183,7 +183,7 @@ set (SRCS
 esma_add_library (${this}
   SRCS ${SRCS}
   DEPENDENCIES fms_r8
-  INCLUDES src/mom5/ocean_param/gotm-4.0/include src/mom5/ocean_core
+  INCLUDES src/mom5/ocean_param/gotm-4.0/include src/mom5/ocean_core ${INC_NETCDF}
```

Comment:
Is `${INC_NETCDF}` variable set by `FindNetCDF.cmake`?  I think the convention is `${NETCDF_INCLUDE_DIRS}`

Reply:
Unfortunately, we currently aren't using `FindNetCDF.cmake`, but rather sort of "hard coding" from `FindBaselibs.cmake`. If/When we move to `FindNetCDF.cmake` we will need to change. (Note our system has a raft of non-standard CMake that I've learned.)

Reply:
Since we are including the ecbuild macros, we have access to FindNetCDF.cmake. It is then possible to use that in our FinBaselibs.cmake. 
If this is a change that will trickle to many repos, then keeping it the way it is, is fine.
Sooner or later, that change will be needed.